src/pages/manualLoggingSteps/manualLoggingSteps.tsx

```diff
@@ -0,0 +1,155 @@
+import React, { useState } from 'react';
+import {
+    IonButton,
+    IonCol,
+    IonContent,
+    IonHeader,
+    IonInput,
+    IonItem,
+    IonLabel,
+    IonNote,
+    IonRouterLink,
+    IonTitle,
+    IonToolbar,
+    IonGrid,
+    IonRow,
+} from '@ionic/react';
+//import ExploreContainer from '../../components/ExploreContainer';
+import './manualLoggingSteps.css';
+
+const ManualSteps: React.FC = () => {
+    const [username, setUsername] = useState();
+    const [password, setPassword] = useState();
+    const [isTouched, setIsTouched] = useState(false);
+    const [isValid, setIsValid] = useState<boolean>();
+    var stepsToLog: number;
+    var date: Date;
+    // var pastRecordDates: Array: Date;
+    var pastRecordDates: Array<Date>;
+    var pastRecordSteps: Array<number>;
+    interface Record {
+        time: Date;
+        stepsWaled: number;
+    }
+    var pastRecords: Array<Record>;
+
+    function writeToDatabase() {
+        if (stepsToLog > 0) {
+            console.log(stepsToLog);
+            console.log(date);
+        }
+    }
+
+    function storeSteps(ev: Event) {
+        const steps: number = Number((ev.target as HTMLInputElement).value);
+        if (steps <= 0) {
+            stepsToLog = 0;
+            return false;
+        }
+        stepsToLog = steps;
+    }
+
+    function storeNewDate(ev: Event) {
+        const newDate = new Date((ev.target as HTMLInputElement).value);
+        date = newDate;
+        /*     
+            Could implement error checking to ensure date is not beyonde today, and give time limit to how far in past
```

Comment:
As I understand it, they definitely want a restriction for how far in the past you can log. I'll add it to my questions for the sponsors sheet.

Reply:
On second thought, maybe this should be a configurable setting? It seems like something they might change their mind on...

Reply:
I can make there to be a check for greater than date and less than. Aka date bounds checking with some set varialbes and make something to pull in the values from the DB.